services/horizon/internal/ingest/processors/claimable_balances_change_processor.go

```
@@ -53,59 +58,32 @@ func (p *ClaimableBalancesChangeProcessor) ProcessChange(ctx context.Context, ch
 
 func (p *ClaimableBalancesChangeProcessor) Commit(ctx context.Context) error {
 	var (
-		cbsToUpsert   []history.ClaimableBalance
+		cbsToInsert   []history.ClaimableBalance
 		cbIDsToDelete []string
 	)
 	changes := p.cache.GetChanges()
 	for _, change := range changes {
-		switch {
-		case change.Pre == nil && change.Post != nil:
+		if change.Post != nil {
```

can you return an error if there is ledger entry change update?

---

done